jsonnetfile.json

```diff
@@ -1,5 +1,15 @@
 {
   "version": 1,
-  "dependencies": [],
+  "dependencies": [
```

Comment:
Why are you adding gocd-jsonnet as a dependency of itself?

Reply:
I was just following the readme:

```
jb install https://github.com/getsentry/gocd-jsonnet.git/libs@v1.4.2
```

It does seem wrong though.

Reply:
The README is geared to "This is how you use this library", not "This is everything you need to do to build this library"

Reply:
You could add a jsonnetfile.json to the example if you wanted to have an example project that builds from start to finish?